dm/_utils/terror_gen/errors_release.txt

```diff
@@ -130,6 +130,7 @@ ErrMetadataNoBinlogLoc,[code=11123:class=functional:scope=upstream:level=low], "
 ErrPreviousGTIDNotExist,[code=11124:class=functional:scope=internal:level=high], "Message: no previous gtid event from binlog %s"
 ErrNoMasterStatus,[code=11125:class=functional:scope=upstream:level=medium], "Message: upstream returns an empty result for SHOW MASTER STATUS, Workaround: Please check the upstream settings like privileges, RDS settings to read data from SHOW MASTER STATUS."
 ErrBinlogNotLogColumn,[code=11126:class=binlog-op:scope=upstream:level=high], "Message: upstream didn't log enough columns in binlog, Workaround: Please check if session `binlog_row_image` variable is not FULL, restart task to the location from where FULL binlog_row_image is used."
+ErrShardDDLOptimismAddNotFullyDroppedColumn,[code=11127:class=functional:scope=internal:level=medium], "Message: fail to resolve add column for optimistic shard ddl lock %s: %s, Workaround: Please use `binlog skip` command to skip this error."
```

Comment:
Maybe we can add more description for it. e.g. Added new col that has not been completely deleted, which may cause data inconsisdent.